extensions/amp-subscriptions-google/amp-subscriptions-google.md

```diff
@@ -145,7 +145,9 @@ Where `data` matches the [entitlements response](https://github.com/subscription
           }
         },
         {
-          "serviceId": "subscribe.google.com"
+          "serviceId": "subscribe.google.com",
+          "hasAssociatedAccountUrl": "https://...",
```

Comment:
Might need to bikeshed on this name a bit.

Reply:
let me know if you have suggestions